libs/ui/src/printed_ballot.tsx

```diff
@@ -237,6 +240,9 @@ interface Props {
   votes: VotesDict;
 }
 
+/**
+ * Renders a printed ballot, generated by a ballot-marking device
+ */
 export function PrintedBallot({
```

Comment:
This does make me wonder whether I should rename this component now that it lives outside of `frontends/bmd`, to distinguish it from a printed HMPB. Maybe `BMDPrintedBallot`?

Reply:
Went forth with that change! https://github.com/votingworks/vxsuite/pull/1736/commits/ce66a5c93cb4a70e644dd898daac083113bdefb1

Reply:
Yeah I was thinking the same thing right before encountering this comment!